chroot/run.go

```diff
@@ -1107,7 +1107,12 @@ func setupChrootBindMounts(spec *specs.Spec, bundlePath string) (undoBinds func(
 		}
 		subSys := filepath.Join(spec.Root.Path, m.Mountpoint)
 		if err := unix.Mount(m.Mountpoint, subSys, "bind", sysFlags, ""); err != nil {
-			logrus.Warningf("could not bind mount %q, skipping: %v", m.Mountpoint, err)
+			msg := fmt.Sprintf("could not bind mount %q, skipping: %v", m.Mountpoint, err)
+			if strings.HasPrefix(m.Mountpoint, "/sys") {
+				logrus.Infof(msg)
+			} else {
+				logrus.Warningf(msg)
```

Comment:
Having just checked the `logrus` docs, I'm finding myself baffled that it doesn't offer `logrus.Log(variable_level, msg)`. 

Aside from this beyond-your-control nit, your logic looks sound to me. Thanks.

Reply:
you could use something like I've done in https://github.com/containers/podman/pull/8726/files#diff-c87419775e693e4c4cc2eded56039e4231b06baa571342c1eb0b19f439b6b1ddR1440-R1458

but we can refactor it later

Reply:
That does not look a hell of a lot cleaner then what I did.